### PR TITLE
Joy reorg

### PR DESCRIPTION
**Pull request by pm0kjp**
Here I add "how to create a module and who does what in terms of GitHub".  Give it a read and decide if it's clear enough.  I've pulled over some materials from our other repo.

### CODE REVIEW

**Comment by pm0kjp** [comment]
Hi all, please check and approve if you agree! (really approve, don't just leave a comment)

**Review by rosemm — CHANGES_REQUESTED** [review]

File: a_sample_module_template/a_sample_module_template.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+Here's one that's just one line long:
+
+[[___]]
+
+Here's a multiple choice with no correct answer. What is your favorite Beatles album?
+
+[(rev)] Revolver
+[(wa)] The While Album
+[(ar)] Abbey Road
+[(help)] Help
```
Comment (franzenr) [inline a_sample_module_template/a_sample_module_template.md:255]: I briefly forgot that "Help!" was one of the album names and not just song title and for a short moment thought that help was a liascript feature that I wasn't aware of yet that functions somewhat like a hint... :joy:

I don't think this is inherently a problem with listing that as one of the four albums, I just thought it was funny, _but_ we could change it to one of their many others if we think enough other people might have the same "Doh!" moment as me.

File: how_to.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+</details>
+
+<details>
+<summary>Scope and Style </summary><br/>
+
+A module in this project will be brief (one hour or less) and therefore of limited, well-described scope.  The topics we present are often complex and
+
+</details>
+
```
[the remark this answers is not in the document]
Comment (pm0kjp) [reply how_to.md:46]: haha yep I will remove!

File: how_to.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+## Checklists and Reports
+
+### Inclusivity Guidelines
+
+This is a working document articulating our goals and standards for creating content that will be valuable and accessible to as wide a range of users as possible. Specifically, we are designing with a variety of ["edge" users](https://guide.inclusivedesign.ca/activities/inclusivedesignmapping) in mind: those with limited access to visual content or auditory content, those with barriers related to attention, cognition, sensory processing, or language, and those with limited technology access and/or financial resources. A core principle is to allow users as much flexibility as possible to configure their learning experiences to meet their own needs and preferences.
+
+The guidelines presented here are inspired by several other guides including the [Web Content Accessibility Guidelines WCAG21](https://www.w3.org/WAI/WCAG21/quickref) and [The Inclusive Learning Design Handbook](https://handbook.floeproject.org/approachesoverview). Recommendations specific to educational videos can be found [here](https://ctl.wiley.com/how-to-ensure-accessibility-for-educational-videos/).
```
Comment (franzenr) [inline how_to.md:79]: broken link

File: how_to.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+</details>
+
+<details>
+<summary>Inclusive Design</summary><br/>
+
+We expect authors to design with a variety of ["edge" users](https://guide.inclusivedesign.ca/activities/inclusivedesignmapping) in mind: those with limited access to visual content or auditory content, those with barriers related to attention, cognition, sensory processing, or language, and those with limited technology access and/or financial resources.  Wherever possible we encourage a multi-modal approach to education, such that no instruction relies solely on a single type of communication (text, video, audio, images, code) but provides several ways to engage with materials.  Our [inclusivity guidelines can be found below](#Inclusivity_Guidelines).
```
Comment (franzenr) [inline how_to.md:27]: broken link

File: how_to.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+
+Once any outstanding improvements are addressed, the administrator checking your module will approve the PR with a squash and merge and delete your branch.  Congratulations, your module is now part of the portfolio of educational modules for the project!
+
+## Checklists and Reports
+
+### Inclusivity Guidelines
+
+This is a working document articulating our goals and standards for creating content that will be valuable and accessible to as wide a range of users as possible. Specifically, we are designing with a variety of ["edge" users](https://guide.inclusivedesign.ca/activities/inclusivedesignmapping) in mind: those with limited access to visual content or auditory content, those with barriers related to attention, cognition, sensory processing, or language, and those with limited technology access and/or financial resources. A core principle is to allow users as much flexibility as possible to configure their learning experiences to meet their own needs and preferences.
```
Comment (franzenr) [inline how_to.md:77]: broken link

File: how_to.md (hunk 3 above)
Comment (franzenr) [reply how_to.md:79]: realize I didn't specify which one... it's the WCAG21 link here.

File: a_sample_module_template/a_sample_module_template.md (hunk 1 above)
Comment (rosemm) [reply a_sample_module_template/a_sample_module_template.md:255]: Oh, good call! I didn't think of that. No reason to risk confusion over a silly example! :)

File: how_to.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+* [ ] Avoids unnecessarily gendered language (e.g. uses "they" singular rather than "he or she" for an unknown person).
+* [ ] Informative link text (e.g. instead of "To learn more about python, click [here](www.example.com)", say "Read this article to [learn more about python](www.example.com).")
+* [ ] Includes accurately formatted and functional link to feedback form.
+
+#### Organization
+
+* [ ] Clear, informative headers and sensible hierarchical structure (the TOC in the left margin should give a good overview of the content convered)
+* [ ] Adheres to the module template structure
+* [ ] Uses specially formatted highlight boxes consistently and appropriately
+* [ ] Short, digestable pieces --- avoids long paragraphs and break long sections up with sub-headers
```
Comment (franzenr) [inline how_to.md:161]:
```suggestion
* [ ] Short, digestible pieces --- avoids long paragraphs and breaks long sections up with sub-headers
```

File: how_to.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+
+## Formative assessment
+
+* [ ] Frequent formative assessment in the form of knowledge checks and/or hands-on exercises
```
Comment (franzenr) [inline how_to.md:165]:
```suggestion
* [ ] Frequent [formative assessment](https://www.cmu.edu/teaching/assessment/basics/formative-summative.html) in the form of knowledge checks and/or hands-on exercises
```

Formative assessment may not be a familiar term to contributors outside the educational space -- seems like a good time to practice what we preach :wink: 
This is just the first link when I googled formative assessment -- we can always use another if someone has a preferred resource!

**Review by franzenr — CHANGES_REQUESTED** [review]
ah yes, now I see how I could have done all of those comments at once instead of sending off notifications each time ... Sorry about that! 😬

**Review by pm0kjp — COMMENTED** [review]
GREAT and thorough review @franzenr TY!

**Comment by rosemm** [comment]
I pushed changes in response to Rose F's suggestions (thank you!). I didn't change the WCAG21 link because it appears that w3.org is down at the moment, and I feel like that has to be temporary.

**Review by leemc-data-ed — CHANGES_REQUESTED** [review]
Just the one comment in how_to.md, about some sections being hidden in a drop-down. Otherwise lgtm!

File: how_to.md [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+
+A module in this project will be brief (one hour or less) and therefore of limited, well-described scope.  Consider not only what you teach about, but what you intentionally exclude.  It may be useful to map a plan of 2, 3, or more modules that teach related skills and together provide broad coverage, but remember that modules are freestanding.  Modules can describe pre-requisite skills, but cannot ask learners to have acquired the necessary skills from other modules.  For example, a module covering the use of n-grams in NLP should not include a pre-requisite such as "previous completion of the module Introduction to NLTK in Python", but could include "previous experience with NLTK and the ability to ingest text data in Python", perhaps with a link to a related module for learners who would like to acquire such experience there.
+
+## Begin Writing
```
[the remark this answers is not in the document]
Comment (rosemm) [reply how_to.md:40]: Good catch! I just fixed it.

**Review by rosemm — APPROVED** [review]

**Review by leemc-data-ed — APPROVED** [review]

**Review by franzenr — APPROVED** [review]

**Pull request merged**